### PR TITLE
Update timeout of susefirewall2_to_firewalld

### PR DESCRIPTION
**Pull request by coolgw**
Enlarge wait timeout of susefirewall2-to-firewalld -c

- Related ticket:https://progress.opensuse.org/issues/88245
- Needles: na
- Verification run: http://openqa.suse.de/t5390857

### CODE REVIEW

File: lib/services/firewall.pm [hunk 1]
```diff
@@ -30,13 +30,13 @@ sub install_service {
 }
 
 sub susefirewall2_to_firewalld {
-    my $timeout = 180;
+    my $timeout = 360;
     assert_script_run('susefirewall2-to-firewalld -c',                                     timeout => $timeout);
```
Comment (rfan1) [inline lib/services/firewall.pm:34]: Wei, 

Can you pls add some comments why you add $timeout variable here, and why you use 360s?

File: lib/services/firewall.pm [hunk 2]
```diff
@@ -30,13 +30,13 @@ sub install_service {
...
-    assert_script_run('firewall-cmd --permanent --zone=external --add-service=vnc-server', timeout => $timeout);
+    assert_script_run('firewall-cmd --permanent --zone=external --add-service=vnc-server', timeout => 60);
     # On some platforms such as Aarch64, the 'firewalld restart'
     # can't finish in the default timeout.
-    systemctl 'restart firewalld', timeout => $timeout;
-    script_run('iptables -S', timeout => $timeout);
+    systemctl 'restart firewalld', timeout => 60;
```
Comment (rfan1) [inline lib/services/firewall.pm:38]: How about systemctl('restart firewalld', timeout =>60) ?

File: lib/services/firewall.pm (hunk 1 above)
Comment (coolgw) [reply lib/services/firewall.pm:34]: I have not added $timeout variable , this is original code, i just update the timeout value:)

File: lib/services/firewall.pm (hunk 2 above)
Comment (coolgw) [reply lib/services/firewall.pm:38]: Is this coding rule we should using currently?

Comment (rfan1) [reply lib/services/firewall.pm:38]: I don't think so, just a suggestion :)

File: lib/services/firewall.pm (hunk 1 above)
Comment (coolgw) [reply lib/services/firewall.pm:34]: Why 360?
Timeout of susefirewall2-to-firewalld will cost 280s (after check openqa result), so i think 360s currently maybe a good value we can have a try.

Comment (rfan1) [reply lib/services/firewall.pm:34]: Yup, then no problem here.

I'd suggest adding some comment here.

Comment (lemon-suse) [inline lib/services/firewall.pm:34]: Yes, add some comments here will make reviewer easy to understand. I suggest to set '360' here, while '60' set as $timeout and $timeout set to 60,  or just not use this variable.

Comment (lemon-suse) [inline lib/services/firewall.pm:34]: Yes, add some comments here will make reviewer easy to understand. I suggest to set '360' here, while '60' set as $timeout and $timeout set to 60,  or just not use this variable.

**Review by rfan1 — APPROVED** [review]
LGTM

**Comment by hjluo** [comment]
Can you provide a ARM run,cause this mostly happened on the aarch64?  this result will make the PR more convincing.

**Comment by coolgw** [comment]
> Can you provide a ARM run,cause this mostly happened on the aarch64? this result will make the PR more convincing.
let's give a try
https://openqa.nue.suse.com/tests/5572319#

**Comment by coolgw** [comment]
> Can you provide a ARM run,cause this mostly happened on the aarch64? this result will make the PR more convincing.

rerun with timeout=1200 for aarch platform, the result show pass
/usr/share/openqa/script/openqa-clone-custom-git-refspec https://github.com/coolgw/os-autoinst-distri-opensuse/tree/firewalld http://openqa.suse.de/tests/5665613 _GROUP=0
Created job #5686179: sle-15-SP3-Regression-on-Migration-from-SLE12-SPx-aarch64-Build161.1-offline_sles12sp5_pscc_sdk-tcm-wsm_all_full@aarch64 -> http://openqa.suse.de/t5686179

**Comment by hjluo** [comment]
> > Can you provide a ARM run,cause this mostly happened on the aarch64? this result will make the PR more convincing.
> 
> rerun with timeout=1200 for aarch platform, the result show pass
> /usr/share/openqa/script/openqa-clone-custom-git-refspec https://github.com/coolgw/os-autoinst-distri-opensuse/tree/firewalld http://openqa.suse.de/tests/5665613 _GROUP=0
> Created job #5686179: sle-15-SP3-Regression-on-Migration-from-SLE12-SPx-aarch64-Build161.1-offline_sles12sp5_pscc_sdk-tcm-wsm_all_full@aarch64 -> http://openqa.suse.de/t5686179

Look  good to me and thanks for the logs

**Pull request merged**